ios/FluentUI/Divider/DividerTokens.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+// MARK: - DividerStyle
+
+/// Pre-defined styles of the divider
+@objc public enum MSFDividerStyle: Int {
+    case `default`
+    case shadow
```

Comment:
The spec didn't have a shadow style, but the previous separator did, so I added one.

Reply:
This would be another great thing to discuss with design. It makes sense to keep the compatibility now, but if the designers want something else, this is the best time to make more significant changes.
The name of the style being "shadow" is also a bit misleading in my opinion, since it doesn't actually project any shadow.